robotbase/src/main/java/com/team2357/frc2023/subsystems/ArmRotationSubsystem.java

```diff
@@ -0,0 +1,45 @@
+package com.team2357.frc2023.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.team2357.frc2023.Constants;
```

Comment:
I would be fine with updating the import paths for the subsystem constants to be `...Constants.Subsystem.*` so we do not need to do `Constants.Subsystem` for every constants.